bors: revert to checking Github CI

otan opened the pull request:
It makes Github statuses complicated.

Release justification: non-production code change

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/53528)
<!-- Reviewable:end -->

otan committed:
bors: revert to checking Github CI

It makes Github statuses complicated.

Release justification: non-production code change

Release note: None

otan commented:
bors r+

let's unblock everyone!

otan commented:
bors r+ p=1

otan commented:
bors r-

otan commented:
bors r+ p=1
